plugins/channel_header_plug/channel_header_plug.tsx

```diff
@@ -319,6 +323,10 @@ class ChannelHeaderPlug extends React.PureComponent<ChannelHeaderPlugProps, Chan
     }
 
     render() {
+        if (this.props.appBarEnabled) {
```

Comment:
Any chance we can control this on the parent, so this component doesn't even mount if the AppBar is enabled?